sjb/config/test_cases/test_pull_request_openshift_ansible_logging.yml

```diff
@@ -1,3 +1,4 @@
 ---
 parent: 'test_cases/test_branch_openshift_ansible_logging.yml'
-overrides: {}
\ No newline at end of file
+overrides:
+  evars: "-e skip_sanity_checks=true"
```

Comment:
So the parent has ` +  evars: "-e skip_sanity_checks=true -e openshift_disable_check=* -e openshift_install_examples=false" ` so this child will remove some of those? why?